src/coreclr/minipal/minipal.h

```diff
@@ -75,6 +75,41 @@ class VMToOSInterface
     // Return:
     //  true if it succeeded, false if it failed
     static bool ReleaseRWMapping(void* pStart, size_t size);
+
+    // Create a template for use by AllocateThunksFromTemplate
+    // Parameters:
+    //  pImageTemplate    - Address of start of template in the image for coreclr. (All addresses passed to the api in a process must be from the same module, if any call uses a pImageTemplate, all calls MUST)
+    //  templateSize      - Size of the template
+    //  codePageGenerator - If the system is unable to use pImageTemplate, use this parameter to generate the code page instead
+    //
+    // Return:
+    //  NULL if creating the template fails
+    //  Non-NULL, a pointer to the template
+    static void* CreateTemplate(void* pImageTemplate, size_t templateSize, void (*codePageGenerator)(uint8_t* pageBase, uint8_t* pageBaseRX, size_t size));
+
+    // Indicate if the AllocateThunksFromTemplate function respects the pStart address on this platform
```

Comment:
It was not clear to me first what the pStart address means here. I have then noticed it is the argument of the function below. For better clarity, I'd say "pStart address passed to AllocateThunksFromTemplate" or something along those lines.